production/training/docker/Dockerfile.Redhat

```diff
@@ -0,0 +1,73 @@
+FROM registry.access.redhat.com/rhel7/rhel
+
+WORKDIR /home/downloads
+
+MAINTAINER Almond Admin <root@almond.stanford.edu>
+
+# args can be set using --build-args, e.g.:
+# --build-arg ALMOUND_CLOUD_VERSION=v1.4.0
+ARG ALMOND_CLOUD_VERSION=v1.5.1
+
+# add redhat repos
+RUN yum-config-manager \
+	--enable rhel-server-rhscl-7-rpms  \
+	--enable rhel-7-server-optional-rpms 
+
+# install basic tools
+RUN yum -y install  wget curl vim net-tools
+
+# add epel repo 
+RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+RUN rpm -ivh epel-release-latest-7.noarch.rpm
+
+# install python3
+RUN yum -y install @development \
+	rh-python36 \
+	rh-python36-numpy \
+	rh-python36-scipy \
+	rh-python36-python-tools  \
+	rh-python36-python-six \
+	rh-python36-PyYAML \
+        python36-pip \
+	python36-devel
+
+RUN scl enable rh-python36 bash
```

Comment:
Since recently, EPEL includes python36 without scl, which is better because it installs in /usr/bin/python3.

Reply:
sure, will switch to EPEL python36.